values-it/strings.xml

```diff
@@ -1,142 +1,181 @@
+            <string name="appintro_terms_explain">I pacchetti precompilati da OpenGApps.org sono forniti SOLO per gentile concessione di OpenGApps.org senza garanzia di ALCUN tipo, a condizione che vengano usati liberamente solo per uso personale, e non è permesso distribuire copie al pubblico se non attraverso OpenGApps.org.\n\nTieni a mente che OpenGApps non ti fornisce alcuna licenza per gli APK di Google inclusi nel pacchetto. I pacchetti Open GApps forniscono soltanto un modo veloce per caricare manualmente gli APK nel dispositivo. È tua responsabilità ottenere le autorizzazioni necessarie, per esempio acquistando un dispositivo con licenza OHA con le Google Apps preinstallate e/o scaricare le applicazioni dal Play Store di Google.</string>
         <!--HINTS-->
-            <string name="slide_arch_hint">L’app rileva automaticamente l’architettura del tuo dispositivo</string>
-            <string name="slide_android_hint">L’app rileva automaticamente la versione di Android del tuo dispositivo</string>
-            <string name="slide_variant_hint">L’app rileva automaticamente la variante installata sul tuo dispositivo</string>
+            <string name="slide_arch_hint">L’app rileva automaticamente l’architettura del dispositivo</string>
+            <string name="slide_android_hint">L’app rileva automaticamente la versione di Android del dispositivo</string>
+            <string name="slide_variant_hint">L’app rileva automaticamente la variante installata nel dispositivo</string>
         <!--VARIOUS-->
             <string name="detected">rilevato</string>
 
     <!--______PREFERENCES______-->
         <!--LABELS-->
+            <string name="pref_header_interface">Interfaccia</string>
             <string name="pref_header_install">Installazione</string>
             <string name="pref_header_download">Download</string>
-            <string name="new_folder">Nuova Cartella</string>
+            <string name="label_dark_mode">tema scuro</string>
+            <string name="new_folder">Nuova cartella</string>
             <string name="pref_header_gapps_selection">Seleziona GApps</string>
             <string name="action_settings">Impostazioni</string>
             <string name="label_wifi_only">Solo Wi-Fi</string>
-            <string name="label_download_dir">Cartella di Download</string>
-            <string name="label_wipe_cache">Cancella Cache/Dalvik</string>
-            <string name="label_download_md5">Scarica MD5</string>
-            <string name="only_newest_version">Pulisci automaticamente i download</string>
-            <string name="label_download_versionlog">Scarica il Log di Versione</string>
-            <string name="label_root_mode">Usa Permessi di Root</string>
+            <string name="label_download_dir">Cartella download</string>
+            <string name="label_wipe_cache">Azzera cache/dalvik</string>
+            <string name="label_download_md5">Download MD5</string>
```

Comment:
Maybe it's better to keep "Scarica" here, since it's a directive, and not an action like the "Download" button